conv.py

```diff
@@ -1,6 +1,14 @@
+import math
+
+
 def celsius_to_fahrenheit(celsius: float) -> float:
     return celsius * 9 / 5 + 32
 
 
 def mbar_to_inhg(mbar: float) -> float:
     return mbar * 0.02952998751
+
+
+def absolute_humidity_g_m3(temp_celsius: float, relative_humidity_pct: float) -> float:
```

Comment:
Please add some test cases for this function that are easily human-readable for a quick sanity check.